src/reference/modules/usage/pages/variables.adoc

```diff
@@ -1051,6 +1051,8 @@ Iterations:
 
 ----
 
+For a full example of file template edition, see: https://docs.rudder.io/rudder-by-example/current/files/advanced-file-templating.html
```

Comment:
Read a xref:rudder-by-example:ROOT:files/advanced-file-templating.adoc[full example] of file template in _Rudder by example_.